src/util-frontend.js

```diff
@@ -213,3 +213,96 @@ export function getToastErrorTimeout() {
 
     return errorTimeout;
 }
+
+class RelativeTimeFormatter {
+    /**
+     * Default locale and options for Relative Time Formatter
+     */
+    constructor() {
+        this.locale = currentLocale();
```

Comment:
As used, I don't think we need to store the locale.
It is always directly used to construct the `RelativeTimeFormat` below

=> Let's make this unit a local variable instead

Reply:
Right, refactored to get locale while initializing instance.